crypto.go

```diff
@@ -30,10 +30,13 @@ type dhScheme interface {
 	Unmarshal(enc []byte) (KEMPublicKey, error)
 	DH(priv KEMPrivateKey, pub KEMPublicKey) ([]byte, error)
 	PublicKeySize() int
+	setEphemeralKeyPair(sk KEMPrivateKey, pk KEMPublicKey)
 }
 
 type dhkemScheme struct {
 	group dhScheme
+	skE KEMPrivateKey
```

Comment:
It would be helpful to add a comment explaining these, since it might not be totally obvious.

Reply:
Yep, will do!